app/controllers/items_controller.rb

```diff
@@ -0,0 +1,35 @@
+class ItemsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_wishlist!, only: %i(new create)
+
+  def new
+    @item = @wishlist.items.new
+    authorize! :new, @item
+  rescue CanCan::AccessDenied
+    redirect_to @wishlist, alert: t("generic_error")
+  end
+
+  def create
+    @item = @wishlist.items.create(item_params)
+    authorize! :create, @item
+    @item.save!
```

Comment:
`create = new + save`
So writing `new` in the first line and save! would be a valid option.